packages/Frontend/src/components/Authenticate.jsx

```diff
@@ -62,7 +79,7 @@ export default function Authenticate({
         )}
         { error && (
           <Message negative>
-            <p>{error}</p>
+            <p>{handleErrorMessage(error)}</p>
```

Comment:
This calls that switch case function above in order to get a nice human readable message.